src/components/HrefActions.vue

```diff
@@ -115,7 +120,7 @@ export default {
       }
     },
     actions() {
-      return Object.entries(this.isAsset ? AssetActions : LinkActions)
+      return Object.entries(this.isAsset ? AssetActions : this.isItem ? ItemActions : LinkActions)
```

Comment:
This is a little to hard to read, at least add ( ) around, please. Or move the condition above the return statement.